pkg/apiutil/serverapi/middleware.go

```diff
@@ -145,21 +146,21 @@ func (p *customReverseProxies) ServeHTTP(w http.ResponseWriter, r *http.Request)
 
 		resp, err := p.client.Do(r)
 		if err != nil {
-			log.Error("request failed", zap.Error(err))
+			log.Error("request failed", zap.Error(errs.ErrHTTPRequest.FastGenByArgs()), zap.NamedError("cause", err))
```

Comment:
We need to wait for the `pingcap/errors` to do some changes to avoid the two errors.